marketplace-image.json

```diff
@@ -59,7 +59,9 @@
         "bundle add workarea",
         "bundle exec rails generate workarea:install",
         "bundle exec rails assets:precompile",
-        "echo \"SECRET_KEY_BASE=$(bundle exec rails secret)\" >> /etc/workarea.env"
+        "echo \"SECRET_KEY_BASE=$(bundle exec rails secret)\" >> /etc/workarea.env",
+        "rm /etc/nginx/sites-available/default",
```

Comment:
This is because APT refused to install the nginx package since a file that it was supposed to write was already in there...that's why we have to jump through this hoop right here.